### PR TITLE
Use correct name scheme for Mac release

### PR DESCRIPTION
**Pull request by ageorgou**
Making the reported name match the actual file.

I tried replacing the instances of `<version>` in the instructions with `{{stable_release}}` and `{{stable_release_short}}`, but it doesn't render correctly inside backticks. Is it okay to use a `<code>` block as with the Windows path above?

### CODE REVIEW

File: downloads/platform.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Uninstallation is preferably performed by using the Windows uninstaller. The dir
 
 ## macOS
 
-On macOS, a `Julia-<version>.dmg` file is provided, which contains `Julia-<version>.app`. Installation is the same as any other Mac software: drag the `Julia-<version>.app` to Applications Folder's Shortcut. The Julia download runs on macOS 10.9 Mavericks and later releases. You can build from source for macOS 10.6 Snow Leopard (possibly earlier versions as well) and 32-bit but neither are fully supported.
+On macOS, a ~~~<code>julia-{{stable_release}}-mac64.dmg</code>~~~ file is provided, which contains ~~~<code>Julia-{{stable_release_short}}.app</code>~~~. Installation is the same as any other Mac software: drag the ~~~<code>Julia-{{stable_release_short}}.app</code>~~~ to Applications Folder's Shortcut. The Julia download runs on macOS 10.9 Mavericks and later releases. You can build from source for macOS 10.6 Snow Leopard (possibly earlier versions as well) and 32-bit but neither are fully supported.
```
Comment (fredrikekre) [inline downloads/platform.md:62]:
```suggestion
On macOS, a ~~~<code>Julia-{{stable_release}}-mac64.dmg</code>~~~ file is provided, which contains ~~~<code>Julia-{{stable_release_short}}.app</code>~~~. Installation is the same as any other Mac software: drag the ~~~<code>Julia-{{stable_release_short}}.app</code>~~~ to Applications Folder's Shortcut. The Julia download runs on macOS 10.9 Mavericks and later releases. You can build from source for macOS 10.6 Snow Leopard (possibly earlier versions as well) and 32-bit but neither are fully supported.
```

Comment (giordano) [reply downloads/platform.md:62]: As confusing as it is, dhe `.dmg` is lowercase, the `.app` is uppercase: https://julialang-s3.julialang.org/bin/mac/x64/1.6/julia-1.6.0-mac64.dmg

Comment (fredrikekre) [reply downloads/platform.md:62]: Ah, so it was wrong before.

Comment (ageorgou) [reply downloads/platform.md:62]: Yes, the capitalisation and missing `-mac64` are what I tried to fix, then thought it should also use the version variables.

Comment (fredrikekre) [reply downloads/platform.md:62]: Yea, thanks.

**Pull request merged**